modules/books.js

```diff
@@ -0,0 +1,21 @@
+export default class Books {
+  constructor() {
+    if (localStorage.getItem('books') === null) {
+      this.books = [];
+    } else {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
+  }
+
+  addBook = (book) => {
+    this.books.push(book);
+
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  removeBook = (bookIndex) => {
+    this.books.splice(bookIndex, 1);
```

Comment:
- [ ] [OPTIONAL] Please kindly prefer methods that do not directly mutate the array. You can use `concat` and `filter` in place of `push` and `splice`